merlin/systems/dag/op_runner.py

```diff
@@ -30,14 +33,21 @@ def __init__(self, config, repository="./", version=1, kind=""):
             op_module = importlib.import_module(module_name)
             op_class = getattr(op_module, class_name)
 
-            operator = op_class.from_config(op_config)
+            operator = op_class.from_config(
```

Comment:
@karlhigley In order to load an artifact required by an operator, passing the `model_*` related params through to all ops that use the op runner here. I'm guessing this may have considered before because the existing OperatorRunner accepts a model repository and version (though currently unused). Does this seem like a reasonable thing to do?